### ISSUE TITLE
aws-ecs: `ContainerImage.fromEcrRepository` should be `tagOrDigest`

### ISSUE THREAD

**Issue by cheruvian** [aws/aws-cdk#30463]
### Describe the issue

The underlying `EcrImage` constructor has the correct naming `tagOrDigest` since most use the helper static method the name and documentation should be updated there is as well.

https://github.com/aws/aws-cdk/blob/v2.144.0/packages/aws-cdk-lib/aws-ecs/lib/images/ecr.ts#L22

### Links

https://github.com/aws/aws-cdk/blob/v2.144.0/packages/aws-cdk-lib/aws-ecs/lib/container-image.ts#L22
https://github.com/aws/aws-cdk/blob/v2.144.0/packages/aws-cdk-lib/aws-ecs/lib/images/ecr.ts#L22

**Comment by pahud** [comment]
Yes, but I am not sure how to fix that without breaking changes. Any ideas?

**Comment by cheruvian** [comment]
Looks like it's a positional argument not a named one, so shouldn't be a breaking change? (unless this is some breaking change quirk of JSII?)
